Modules/_functoolsmodule.c

```diff
@@ -764,8 +765,6 @@ lru_cache_make_key(PyObject *args, PyObject *kwds, int typed)
         Py_INCREF(args);
         return args;
     }
-
-    kwds_size = kwds ? PyDict_GET_SIZE(kwds) : 0;
     assert(kwds_size >= 0);
```

Comment:
This assert looks useless. Off course the dict size >= 0. Perhaps it is remnants from using `PyDict_Size()` instead of `PyDict_GET_SIZE()`. But `PyDict_GET_SIZE()` never returns negative number, and it contains an assert inside which checks the type of the argument.

Reply:
Agreed.